aws_lambda_powertools/logging/utils.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import Callable, List, Optional, TypeVar
+
+from .logger import Logger
+
+PowertoolsLogger = TypeVar("PowertoolsLogger", bound=Logger)
+
+
+def copy_config_to_registered_loggers(
+    source_logger: PowertoolsLogger,
+    exclude: Optional[List[str]] = None,
+    include: Optional[List[str]] = None,
+) -> None:
+    """Enable powertools logging for imported libraries.
+
+    Attach source logger handlers to external loggers.
+    Modify logger level based on source logger attribute.
+    Ensure powertools logger itself is excluded from registered list.
+    """
+
```

Comment:
Could you re-add the algorithm as a comment? This will help anyone else coming back to this code to understand the multi-nested conditional

Reply:
done